lib/aruba/rspec.rb

```diff
@@ -18,6 +18,12 @@
     if self.class.include? Aruba::Api
       restore_env
       setup_aruba
+
```

Comment:
This whole changing environment-thing in aruba is a bit of a hassle. I would love to this in a different PR, please.